templates/vdcuser/vdcuser.py

```diff
@@ -31,7 +35,8 @@ def ovc(self):
 
     def get_fqid(self):
```

Comment:
well the js ovc client uses the username without the `@provider` so the name saved in this template saves it in this form, fqid returns the username with the `@provider`

Reply:
but if only the fqid would be needed outside of the template, then maybe it should be renamed to `get_name`

Reply:
Ah, I see. I think only name with @provider will be used outside, because all methods like `user_add`/`user_delete` are using the full username, and I don't know where else we need it.

Reply:
Yep, that's why `get_name` is not implemented for vdcuser, haven't needed it so far